package.json

```diff
@@ -235,12 +235,12 @@
     "@opam/merlin-extend": "0.4",
     "rench": "github:bryphe/rench#4860ef4",
     "reason-tree-sitter": "github:onivim/reason-tree-sitter#e2c571e",
-    "revery": "github:revery-ui/revery#758631f",
+    "revery": "glennsl/revery#6a9fbd6",
```

Comment:
This will need to be updated after https://github.com/revery-ui/revery/pull/731 has been merged